### PR TITLE
Feature/mat 5138 validate return types cql changed

### PR DESCRIPTION
**Pull request by nmorasb**
## MADiE PR

Jira Ticket: [MAT-5138](https://jira.cms.gov/browse/MAT-5138)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/cms/gov/madie/measure/services/MeasureService.java [hunk 1]
```diff
@@ -35,12 +81,13 @@ public void checkDuplicateCqlLibraryName(String cqlLibraryName) {
   }
 
   public void validateMeasurementPeriod(Date measurementPeriodStart, Date measurementPeriodEnd) {
-
+    System.out.println("validating measurement period here");
```
Comment (adongare) [inline src/main/java/cms/gov/madie/measure/services/MeasureService.java]: looks like debug statement

File: src/main/java/cms/gov/madie/measure/services/MeasureService.java [hunk 2]
```diff
@@ -35,12 +81,13 @@ public void checkDuplicateCqlLibraryName(String cqlLibraryName) {
...
     if (measurementPeriodStart == null || measurementPeriodEnd == null) {
       throw new InvalidMeasurementPeriodException(
           "Measurement period date is required and must be valid");
     }
 
+    System.out.println("didn't blow up?? why not??");
```
Comment (adongare) [inline src/main/java/cms/gov/madie/measure/services/MeasureService.java]: and this one

Comment (nmorasb) [reply src/main/java/cms/gov/madie/measure/services/MeasureService.java]: removed

File: src/main/java/cms/gov/madie/measure/services/MeasureService.java (hunk 1 above)
Comment (nmorasb) [reply src/main/java/cms/gov/madie/measure/services/MeasureService.java]: removed!

File: src/main/java/cms/gov/madie/measure/utils/MeasureUtil.java [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+   * @return
+   */
+  public Measure validateAllMeasureGroupReturnTypes(Measure measure) {
+    if (measure == null) {
+      return null;
+    }
+
+    final String elmJson = measure.getElmJson();
+    boolean groupsExistWithPopulations = isGroupsExistWithPopulations(measure);
+    Measure.MeasureBuilder measureBuilder = measure.toBuilder();
```
Comment (adongare) [inline src/main/java/cms/gov/madie/measure/utils/MeasureUtil.java:41]: 👍

**Review by adongare — APPROVED** [review]
looks good!

**Pull request merged**